docs/current_docs/manuals/developer/python/428321-constructor.mdx

```diff
@@ -1,110 +0,0 @@
----
-slug: /manuals/developer/python/428321/constructor
-displayed_sidebar: "current"
-toc_max_heading_level: 2
-title: "Module Constructor"
----
-
-
-# Module Constructor
-
-
-The main object's constructor is exposed as the entrypoint function for a module.
-Its parameters are available as flags in the `dagger call` command directly.
-
-This is a simple way to accept module-wide configuration, or just to set
-a few attributes without having to create setter functions for them.
-
-In the Python SDK, the [`@dagger.object_type`][@object_type] decorator wraps
-[`@dataclasses.dataclass`][dataclass], which means that an `__init__()` method
-is automatically generated, with parameters that match the declared class attributes.
-
-Here is an example with an object that has typed attributes:
-
-```python file=./snippets/constructor/simple/main.py
-```
-
-And here is an example call for this module:
-
-```shell
-dagger call --name=Daggerverse message
-```
-
-The result will be:
-
-```shell
-Hello, Daggerverse!
-```
-
-Constructor arguments are documented through their attribute documentation,
```

Comment:
This inline doc section is not transferred because there is a similar example (object documentation) already provided in the inline documentation page, but lmk if this is not correct @helderco